frontend/src/components/Chat.jsx

```diff
@@ -70,4 +84,19 @@ function Chat(props) {
   );
 }
 
+const MessageChat = styled.div`
+  display: flex;
+  justify-content: space-between;
+`
+
+const DeleteButton = styled.button`
```

Comment:
Can you add `background-color: transparent;` property to this button? Because I see that it has a grey background